web/templates/left_sidebar.hbs

```diff
@@ -1,6 +1,6 @@
 <div class="left-sidebar" id="left-sidebar" role="navigation">
-    <div class="narrows_panel">
-        <ul id="global_filters" class="filters">
+    <div id="left_sidebar_navigation_area">
+        <ul id="views" class="filters">
```

Comment:
I don't think an ID as short as `views` is going to make us happy. How about `left_sidebar_navigation_list` for this element, since it's the list for that area.